user/src/components/FormList/type.ts

```diff
@@ -0,0 +1,4 @@
+export type FormItem = {
+  label: string;
+  content: string;
+};
```

Comment:
とりあえず変更不可のプロパティを設定しておけば、表示は別途コンポーネント側で対応できるのかなと思いました。
```suggestion
export type FormItem = {
  label: string;
  content: string;
  editable?: boolean;
};
```
```typescript
{item.editable === false && (
   <span className="text-red-500 ml-6 text-xs">※変更できません</span>
)}
```

Reply:
今日のgm2 mtで決まっただけど
## ラベルずを作る
---
label.ts で，定数を宣言しておく．

こんな感じでlabel部分の定数を置いておいて、labelとdataを結合してFormItemで渡すことになったから、この修正ちょっと厳しいかな？
https://www.notion.so/nutfes-nutmeg/GM2-1cb41f19206380b682d9e6cf7fc710b5?pvs=4

Reply:
よく考えたらオプショナルだし、結合の時に一緒にやればいけるのか。
要確認でいきます！

Reply:
これ変更できないパターンの方が少ないから逆にします